packages/web-component-tester/wct-browser-legacy/package-lock.json

```diff
@@ -57,25 +57,6 @@
 			"resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
```

Comment:
Update the wct-browser-legacy package-lock? Or, uh, un-update it?

Reply:
yeah...